src/eme.js

```diff
@@ -182,15 +182,19 @@ const defaultPlayreadyGetLicense = (url) => (emeOptions, keyMessage, callback) =
   });
 };
 
-const defaultGetLicense = (url) => (emeOptions, keyMessage, callback) => {
+const defaultGetLicense = (keySystemOptions) => (emeOptions, keyMessage, callback) => {
+  const headers = videojs.mergeOptions(
+    {'Content-type': 'application/octet-stream'},
```

Comment:
Since you're now providing options to add headers, any chance you could not include this by default? For whatever reason, our DRM provider doesn't have `Content-Type` in the `Access-Control-Allow-Headers` header.

Reply:
do you think there would be issues in removing it? I've got no major objections unless removing it may break someone.

Reply:
What if we allowed _removing_ headers by setting them to `null` or something, so you could do:

```js
licenseHeaders: {
  'Content-Type': null
}
```
I am hesitant to remove defaults because it could easily break someone.